mapper/cumulocity/c8y_translator_lib/src/json.rs

```diff
@@ -1,113 +1,82 @@
 //! A library to translate the ThinEdgeJson into C8yJson
 //! Takes thin_edge_json bytes and returns c8y json bytes
 //!
+//! # Examples
+//!
 //! ```
-//! use c8y_translator_lib::json::CumulocityJson;
+//! use c8y_translator_lib::json::from_thin_edge_json;
 //! let single_value_thin_edge_json = r#"{
 //!        "time": "2020-06-22T17:03:14.000+02:00",
 //!        "temperature": 23,
 //!        "pressure": 220
 //!     }"#;
-//! let output = CumulocityJson::from_thin_edge_json(
-//!             &String::from(single_value_thin_edge_json).into_bytes());
+//! let output = from_thin_edge_json(single_value_thin_edge_json.as_bytes());
 //! ```
 
 use crate::serializer;
 use chrono::prelude::*;
-use thin_edge_json::{
-    json::{ThinEdgeJson, ThinEdgeValue},
-    measurement::GroupedMeasurementVisitor,
-};
-
-#[derive(Debug, Eq, PartialEq)]
-pub struct CumulocityJson;
+use thin_edge_json::json::{ThinEdgeJsonParserError, *};
```

Comment:
Just use "*". No need for explicit ThinEdgeJsonParserError import.

Reply:
Thanks for catching it.